File: BeeCrowd C++/1013.cpp
#include <iostream>
using namespace std;

int main() {
    int A,B,C;

    cin >> A;
    cin >> B;
    cin >> C;

    int maior = (A+B+abs(A-B))/2;

    if (maior > C){
        cout << maior << " eh o maior" << endl;
    }
    else{
        cout << C << " eh o maior" << endl;
    }
}

File: BeeCrowd Python/Iniciante/1013.py
def main():
    a,b,c = input().split( )
    a = int(a)
    b = int(b)
    c = int(c)
    
    maior_ab = int((a+b+abs(a-b)) / 2)
    
    if maior_ab > c:
        MAIOR = maior_ab
    else:
        MAIOR = c
    
    print(f'{MAIOR} eh o maior')

main()
